style-guide.rst

```diff
@@ -49,7 +47,7 @@ Headings
    * - Underlines with other characters, for example, ``"``, ``#`` or ``*``
      - Further headings
 
-Underlines and overlines must be at least as long as the title or heading.
+Underlines must be at least as long as the title or heading.
```

Comment:
Please let's be more prescriptive here. The underline must be the same length as the header title.

Reply:
I think it should be stated explicitly - it's the most common cause for build failures for developers in my experience.